src/components/modal/modal.jsx

```diff
@@ -0,0 +1,34 @@
+const React = require('react');
+const ReactModal = require('react-modal');
+
+const styles = require('./modal.css');
+
+class ModalComponent extends React.Component {
+    render () {
+        return (
+            <ReactModal
+                className={styles.modalContent}
+                isOpen={this.props.visible}
+                overlayClassName={styles.modalOverlay}
+                ref={m => (this.modal = m)}
+                onRequestClose={this.props.onRequestClose}
+            >
+                <div
+                    className={styles.modalCloseButton}
+                    onClick={this.props.onRequestClose}
+                >
+                    {'x'}
```

Comment:
The only reasons are for editors, explained here https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-literals.md

No reason in this case. We may have had that rule turned on when @tmickel did the first pass of the modal.